docs/DeveloperGuide.md

```diff
@@ -290,31 +290,34 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC1 - List all people saved in the System**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to list all people saved in the System by entering a command.
+2.  System shows a list of persons.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. If there are no contacts stored in the system.
 
-  Use case ends.
+  * 1a1 System informs user that they have no contacts stored.
+  
+    * Use case ends.
```

Comment:
Could change this to just "Use case ends." without the * to be consistent with other use cases.